src/app/app-interactive-map/store/interactive-map.effects.ts

```diff
@@ -261,6 +262,28 @@ export class InteractiveMapEffects {
     })),
   );
 
+  @Effect()
+  incrementRequest: Observable<Action> = combineLatest(
+    this.actions$.pipe(ofType(fromActions.FETCH_SPECIES)),
+    this.actions$.pipe(ofType(fromActions.FETCH_MODELS)),
+    this.actions$.pipe(ofType(fromActions.FETCH_MAPS)),
+  ).pipe(
+    map(() => {
```

Comment:
Please just use `mapTo`, it's more simple https://www.learnrxjs.io/operators/transformation/mapto.html